src/main/java/Duke.java

```diff
@@ -15,57 +16,82 @@ public static void main(String[] args) {
-                taskList[Task.maxTaskNumber] = new Deadline(deadline[0], deadline[1]);
-                Task.maxTaskNumber++;
-                System.out.printf(HorizontalLine + "Deadline added: %s (by: %s)\n" + HorizontalLine, deadline[0], deadline[1]);
+                createDeadline(inputWords);
                 break;
             case "event":
-                String[] event = inputWords[1].split(" /from | /to ", 3);
-                taskList[Task.maxTaskNumber] = new Event(event[0], event[1], event[2]);
-                Task.maxTaskNumber++;
-                System.out.printf(HorizontalLine + "Event added: %s (from: %s to: %s)\n" + HorizontalLine, event[0], event[1], event[2]);
+                createEvent(inputWords);
                 break;
             }
             input = in.nextLine();
         }
         System.out.println(HorizontalLine + "Goodbye!" + "\n" + HorizontalLine);
     }
+
+    private static void createEvent(String[] inputWords) {
+        String[] event = inputWords[1].split(" /from | /to ", 3);
+        taskList[Task.maxTaskNumber] = new Event(event[0], event[1], event[2]);
+        Task.maxTaskNumber++;
+        System.out.printf(HorizontalLine + "Event added: %s (from: %s to: %s)\n" + HorizontalLine, event[0], event[1], event[2]);
+    }
+
+    private static void createDeadline(String[] inputWords) {
+        String[] deadline = inputWords[1].split(" /by ", 2);
+        taskList[Task.maxTaskNumber] = new Deadline(deadline[0], deadline[1]);
+        Task.maxTaskNumber++;
+        System.out.printf(HorizontalLine + "Deadline added: %s (by: %s)\n" + HorizontalLine, deadline[0], deadline[1]);
+    }
+
+    private static void createTodo(String[] inputWords) {
+        taskList[Task.maxTaskNumber] = new ToDo(inputWords[1]);
+        Task.maxTaskNumber++;
+        System.out.println(HorizontalLine + "To do added: " + inputWords[1] + "\n" + HorizontalLine);
+    }
+
+    private static void unmarkTask(String[] inputWords) {
+        int unmarkTaskNumber = Integer.valueOf(inputWords[1]);
+        if (unmarkTaskNumber >= Task.maxTaskNumber) {
+            System.out.println("No such task found\n" + HorizontalLine);
+        } else {
+            taskList[unmarkTaskNumber].unsetDone();
+            System.out.println(HorizontalLine + "Task set as undone: " + taskList[unmarkTaskNumber].getTaskName() +
+                    "\n" + HorizontalLine);
+        }
+    }
+
+    private static void markTask(String[] inputWords) {
+        int markTaskNumber = Integer.valueOf(inputWords[1]);
+        if (markTaskNumber >= Task.maxTaskNumber) {
+            System.out.println("No such task found\n" + HorizontalLine);
+        } else {
+            taskList[markTaskNumber].setDone();
+            System.out.println(HorizontalLine + "Task set as done: " + taskList[markTaskNumber].getTaskName() + "\n"
+                    + HorizontalLine);
+        }
+    }
+
+    private static void displayList() {
```

Comment:
As many before have mentioned before, simple and good use of variable naming. Coding Standard is GREAT!! :)